### PR TITLE
Update/social feed client [✅]

### PR DESCRIPTION
**Pull request by WaDadidou**

### CODE REVIEW

**Review by sakul-budhathoki — APPROVED** [review]
LGTM 👍

**Review by sakul-budhathoki — APPROVED** [review]

**Review by sakul-budhathoki — DISMISSED** [review]

**Review by omniwired — APPROVED** [review]

File: packages/utils/sidebar.ts [hunk 1]
```diff
@@ -20,8 +20,7 @@ import { NetworkKind } from "../networks";
 export const SIDEBAR_LIST: SidebarRecordType = {
   feed: {
     title: "Feed",
-    // route: "Feed",
-    route: "ComingSoon",
```
Comment (omniwired) [inline packages/utils/sidebar.ts:24]: is this on purpose?

**Review by hthieu1110 — APPROVED** [review]
LGTM

**Comment by zxxma** [comment]
LGTM 👍

File: packages/utils/sidebar.ts (hunk 1 above)
Comment (omniwired) [reply packages/utils/sidebar.ts:24]: I guess it was

**Pull request merged**